vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -6,7 +6,7 @@
 #include "MsoUtils.h"
 
 #include <AppModelHelpers.h>
-#include <Base/CoreNativeModules.h>
+#include <Base/CoreNativeModules.h> //TODO: Remove
```

Comment:
For reviewers:

This header was only used to provide Cxx networking modules.
Should we remove it and its corresponding source file altogether?

Reply:
Yup, we can get rid of it then.